<launch>
        <node pkg="teleop_vision" type="camera_capture_flea3" name="camera_capture_flea3" output="screen">
		<param name= "left_image_topic_name" value= "/left_cam" />		
		<param name= "right_image_topic_name" value= "/right_cam" />
		<param name= "frequency" value= "30" />
         </node>


        <node pkg="teleop_vision" type="extrinsic_calib_aruco" name="extrinsic_aruco" output="screen">
        	<param name="cam_intrinsic_calibration_file_path" value="$(find teleop_vision)/resources/LeftCameraParameters.xml" />
		<param name= "camera_image_topic_name" value= "/left_cam" />
		<param name= "board_to_cam_pose_topic_name" value= "/board_to_camera" />
		<param name= "aruco_markers_x" value= "6" />		
		<param name= "aruco_markers_y" value= "4" />		
		<param name= "aruco_marker_length_in_meters" value= "0.0112" />				
		<param name= "aruco_marker_separation_in_meters" value= "0.0022" />				
		<param name= "aruco_dictionary_id" value= "0" />				
         </node>

        <node pkg="teleop_vision" type="overlay" name="overlay" output="screen">
        	<param name="cam_intrinsic_calibration_file_path" value="$(find teleop_vision)/resources/LeftCameraParameters.xml" />
		<param name= "left_image_topic_name" value= "/left_cam" />		
		<param name= "right_image_topic_name" value= "/right_cam "/>
		<param name= "left_cam_pose_topic_name" value= "/board_to_camera "/>
		<param name= "frequency" value= "100" /> 
         </node>

</launch>



